pkg/gce-pd-csi-driver/node.go

```diff
@@ -412,7 +412,7 @@ func (ns *GCENodeServer) NodeGetVolumeStats(ctx context.Context, req *csi.NodeGe
 		}, nil
 	}
 
-	available, capacity, used, inodesFree, inodes, inodesUsed, err := ns.VolumeStatter.StatFS(req.VolumePath)
+	available, capacity, used, inodesFree, inodes, inodesUsed, err := getStatFS(ns.VolumeStatter, req.VolumePath)
```

Comment:
I don't understand why have to to thunk through getStatFS and can't just use the Statter interface directly?

Reply:
you are right, directly use Statter is good. I kind of follow some other calls, which seem have to use this route. 
updated the code. PTAL